classes/class_02/class_02_homework.md

```diff
@@ -0,0 +1,28 @@
+### Homework (Lesson 2)
+
+**Q1)** With just ints and for-loops, recreate this pattern:
+```java
+1
+22
+333
+44444
+555555
+4444
+333
+22
+1
+```
+**Q2)** Write your own switch statement with:
+ * at least one ```break;```
+ * a ```default:``` statement
+
+**Q3)** Write your own if-blocks that use the following at least once:
+* ```!```,```<=``` OR```>=```, ```&&```, ```||```
+* Multiple clauses in one statement.
+* If and else.
+* If, else if, and else.
+
+**Q4)** Use loops to recreate the [Fibonacci Sequence](https://www.mathsisfun.com/numbers/fibonacci-sequence.html) with the initial indexes of 0 and 1.
```

Comment:
Hey @Blunderproof I think we should find a simpler exercise here. Fibonacci might be too complex.